tests/app/dao/test_provider_details_dao.py

```diff
@@ -382,8 +389,12 @@ def test_should_not_error_if_any_provider_in_code_not_in_database(restore_provid
 @freeze_time('2000-01-01T00:00:00')
 def test_update_adds_history(restore_provider_details, sample_provider):
     sample_provider(notification_type=EMAIL_TYPE, identifier=SES_PROVIDER)
-    ses = ProviderDetails.query.filter(ProviderDetails.identifier == SES_PROVIDER).one()
-    ses_history = ProviderDetailsHistory.query.filter(ProviderDetailsHistory.id == ses.id).one()
+
+    stmt = select(ProviderDetails).where(ProviderDetails.identifier == SES_PROVIDER)
+    ses = restore_provider_details.session.scalars(stmt).one()
+
+    stmt = select(ProviderDetailsHistory).where(ProviderDetailsHistory.id == ses.id)
+    ses_history = restore_provider_details.session.scalars(stmt).one()
```

Comment:
```suggestion
    ses_history = restore_provider_details.session.get(ProviderDetailsHistory, ses.id)
```
Could be simplified and stmt line removed.

Reply:
Negative. This would be a behavior change because `one` can raise exceptions. `get` returns an instance or None.